Datenmodell/db_reset.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+MYSQL_USER='homestead'
+MYSQL_PW='secret'
+MYSQL_DB='sqe_bildergalerie'
+VM_SYNC_FOLDER='/vagrant_data'
+
+sudo mysql --user=${MYSQL_USER} --password=${MYSQL_PW} ${MYSQL_DB} < ./db_reset.sql
```

Comment:
Could you please add a line to remove the content of the `uploads` directory

Reply:
Please consider that you won't delete the file `README.md` in the `uploads` directory since the file makes sure that the `upload` folder will be as empty directory in our git repository. Otherwise we could remove the `upload` directory entirely and create it within the `bootstrap.sh` script for vagrant. It's up to you ;-)

One more thing: if you/we decide to keep the `db_reset.sh` shell script please make sure that the working directory will be set to the current directory (where the sql script is available) this would allow to execute the script from a different folder (e.g. `./Datenmodell/db_reset.sh`). And a second thing 🙈 : please check in the script having the execute flag set (`chmod +x ./Datenmodell/db_reset.sh`).

Thanks!